src/win/tty.c

```diff
@@ -163,6 +174,9 @@ int uv_tty_init(uv_loop_t* loop, uv_tty_t* tty, uv_file fd, int readable) {
     /* shared between all uv_tty_t handles. */
     EnterCriticalSection(&uv_tty_output_lock);
 
+    if (uv__vterm_state == UV_UNCHECKED)
```

Comment:
Wouldn't that involve storing the handle somewhere globally since we can not pass an arg to our function with uv_once?